### PR TITLE
solution

### PR DESCRIPTION
**Pull request by uzlss**

### CODE REVIEW

File: app/create_file.py [hunk 1]
```diff
@@ -1 +1,32 @@
...
+    line_args = sys.argv
+    index = 1
+    while index < len(line_args):
+        if line_args[index] == "-d":
+            os.makedirs(os.path.join("app", line_args[index + 1]))
+            os.makedirs(os.path.join("app", line_args[index + 2]))
+            index += 3
+        elif line_args[index] == "-f":
+            create_file(os.path.join("app", line_args[index + 1]))
+            index += 2
```
Comment (ArsenPidhoretskyi) [inline app/create_file.py:29]: Use .index method for list instead of usage while loop

File: app/create_file.py [hunk 2]
```diff
@@ -1 +1,32 @@
...
+        file.write("\n")
+
+
+def read_line() -> None:
+    line_args = sys.argv
+    index = 1
+    while index < len(line_args):
+        if line_args[index] == "-d":
+            os.makedirs(os.path.join("app", line_args[index + 1]))
+            os.makedirs(os.path.join("app", line_args[index + 2]))
```
Comment (ArsenPidhoretskyi) [inline app/create_file.py:25]: Your code will not work if dirs will be more that 2, or 1

**Review by ArsenPidhoretskyi — CHANGES_REQUESTED** [review]

**Review by vsmutok — CHANGES_REQUESTED** [review]
have you tested your code?????


1) executing this command `python create_file.py -d dir1 dir2 -f file.txt` a second time will cause an error
2) you need to create a folder in a folder and in the last folder a file
Screenshot 2023-10-09 at 18 12 16

**Review by vsmutok — CHANGES_REQUESTED** [review]
no need to create `app` folder
image

File: app/create_file.py [hunk 3]
```diff
@@ -1 +1,42 @@
...
+import os
+import sys
+
+
+def create_file(file_path: str) -> None:
+    with open(file_path, "a") as file:
+        file.write(str(datetime.datetime.now().strftime("%Y-%m-%d %H:%M:%S"))
+                   + "\n")
+        line = input("Enter content line: ")
+        index = 1
```
Comment (vsmutok) [inline app/create_file.py:11]: DRY 

here and In 13, 14 lines

File: app/create_file.py [hunk 4]
```diff
@@ -1 +1,42 @@
...
+    if "-f" in line_args:
+        file_name = line_args[line_args.index("-f") + 1]
+        if path:
+            file_path = os.path.join(*path, file_name)
+        else:
+            file_path = os.path.join(file_name)
+        create_file(file_path)
+
+
+read_line()
```
Comment (vsmutok) [inline app/create_file.py:42]: use `if __name__ == '__main__':`

**Review by vsmutok — CHANGES_REQUESTED** [review]

File: app/create_file.py (hunk 3 above)
Comment (uzlss) [reply app/create_file.py:11]: is there any change for "index"?
image

Comment (uzlss) [reply app/create_file.py:11]: for loop i can use "while True", but what's about index?

File: app/create_file.py [hunk 5]
```diff
@@ -1 +1,44 @@
...
+import datetime
+import os
+import sys
+
+
+def create_file(file_path: str) -> None:
+    with open(file_path, "a") as file:
+        file.write(str(datetime.datetime.now().strftime("%Y-%m-%d %H:%M:%S"))
+                   + "\n")
+        i = 1
```
Comment (vasylhnatiuk) [inline app/create_file.py:10]: i is a bad name, rename please

**Review by vasylhnatiuk — CHANGES_REQUESTED** [review]

**Review by vasylhnatiuk — APPROVED** [review]

**Pull request closed without merging**